generators/server/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = JhipsterServerGenerator.extend({
                 this.messageBroker = false;
             }
 
+            this.apiFirst = this.config.get('apiFirst') === 'no' ? false : this.config.get('apiFirst');
```

Comment:
can you use a specific option name something like `enableSwaggerCodeGen` and make it boolean instead of string so that we could just say `swaggerCodeGen: true` and it will be much easier and cleaner especially in EJS